examples/fastapi/main.py

```
@@ -0,0 +1,403 @@
+@app.post("/chat/session", response_model=ChatResponse)
+async def session_chat(request: SessionChatRequest):
+    """
+    세션 기반 채팅 엔드포인트
+    
+    세션을 유지하며 대화 히스토리를 관리합니다.
+    """
+    validate_api_key()
+    
+    try:
+        # LLM 인스턴스 가져오기  
+        llm = get_llm_instance(request.model)
```

Flake8 (7.2.0)

[error] local variable 'llm' is assigned to but never used

(F841)